docs/reference/node/basin.qmd

```diff
@@ -18,10 +18,10 @@ time table, it is empty, or all timestamps of that variable are missing.
 column                | type    | unit                  | restriction
 ---------             | ------- | --------------------- | -----------
 node_id               | Int32   | -                     | sorted
-precipitation         | Float64 | $\text{m}/\text{s}$   | non-negative
-potential_evaporation | Float64 | $\text{m}/\text{s}$   | non-negative
-drainage              | Float64 | $\text{m}^3/\text{s}$ | non-negative
-infiltration          | Float64 | $\text{m}^3/\text{s}$ | non-negative
+precipitation         | Float64 | $\text m}/\text{s}$   | non-negative
```

Comment:
This removes the opening braces.

Reply:
Reverted the commit, and fixed in 3c8633462a30bd5ad9bdbac64ef21902c21ebf6b.
I tried find and replace with regex in VS Code, but couldn't find any matches, probably since the notebooks are JSON encoded. Though if I then searched for the JSON encoding in VS Code that was also buggy, so I did it in trusty Notepad++